vizro-core/src/vizro/models/_components/form/_user_input.py

```diff
@@ -14,22 +14,22 @@
 
 
 class UserInput(VizroBaseModel):
-    """Component provided to `Form` to allow user text input.
+    """Component provided to `Form` to allow single-line user input.
 
     Args:
         type (Literal["user_input"]): Defaults to `"user_input"`.
-        placeholder (str): Default text to display in input field. Defaults to `""`.
-        actions (Optional[List[Action]]): Defaults to `[]`.
         title (str): Title to be displayed. Defaults to `""`.
-        input_type (Literal["text", "number", "password", "email", "search", "tel", "url", "range", "hidden"]):
+        placeholder (str): Default text to display in input field. Defaults to `""`.
+        input_type (Literal["text", "number", "password", "email", "search", "tel", "url"]):
             Type of value to validate user input against. Defaults to `"text"`.
+        actions (Optional[List[Action]]): Defaults to `[]`.
     """
 
     type: Literal["user_input"] = "user_input"
     title: str = Field("", description="Title to be displayed")
```

Comment:
```suggestion
    # TODO: before making public consider naming this field (or giving an alias) label instead of title
    title: str = Field("", description="Title to be displayed")
```